packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -4,23 +4,33 @@ import { error_types, unrecoverable_errors, observer, message_types } from '@der
 import { contract_stages } from '../constants/contract-stage';
 import { switch_account_notification } from '../utils/bot-notifications';
 
+const statistics_storage_key = 'statistics_cache';
+
 export default class RunPanelStore {
     constructor(root_store) {
         this.root_store = root_store;
         this.dbot = this.root_store.dbot;
         this.registerCoreReactions();
+
+        reaction(
+            () => this.statistics,
+            statistics => sessionStorage.setItem(statistics_storage_key, JSON.stringify(statistics))
+        );
     }
 
     run_id = '';
 
-    @observable statistics = {
-        lost_contracts: 0,
-        number_of_runs: 0,
-        total_profit: 0,
-        total_payout: 0,
-        total_stake: 0,
-        won_contracts: 0,
-    };
+    @observable statistics =
+        JSON.parse(sessionStorage.getItem(statistics_storage_key)) !== (undefined || null)
+            ? JSON.parse(sessionStorage.getItem(statistics_storage_key))
+            : {
+                  lost_contracts: 0,
+                  number_of_runs: 0,
+                  total_profit: 0,
+                  total_payout: 0,
+                  total_stake: 0,
+                  won_contracts: 0,
+              };
```

Comment:
Same here, we're calling `JSON.parse(sessionStorage.getItem())` twice, try to reduce it to once.

```JS
        JSON.parse(sessionStorage.getItem(statistics_storage_key)) || {
                  lost_contracts: 0,
                  number_of_runs: 0,
                  total_profit: 0,
                  total_payout: 0,
                  total_stake: 0,
                  won_contracts: 0,
              };
```